android/runtime/common/src/java/org/appcelerator/kroll/KrollRuntime.java

```diff
@@ -114,13 +114,38 @@ public static KrollRuntime getInstance()
 		return instance;
 	}
 
+	/**
+	 * Suggest V8 garbage collection during idle.
+	 */
 	public static void suggestGC()
 	{
 		if (instance != null) {
 			instance.setGCFlag();
 		}
 	}
 
+	/**
+	 * Force V8 garbage collection.
+	 */
+	public static void softGC()
```

Comment:
I'm not sure the naming here is very apt. This looks to be more like `forceJsGC()` and `forceJsAndJavaGC()`